tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -170,30 +187,55 @@ export class WebGPUBackend extends KernelBackend {
     }
   }
 
+  /** Return refCount of a `TensorData`. */
+  refCount(dataId: DataId): number {
+    if (this.tensorMap.has(dataId)) {
+      const tensorData = this.tensorMap.get(dataId);
+      return tensorData.refCount;
+    }
+    return 0;
+  }
+
+  /** Increase refCount of a `TensorData`. */
+  incRef(dataId: DataId): void {
+    const tensorData = this.tensorMap.get(dataId);
```

Comment:
what about add an existence check here?
`if (this.tensorMap.has(dataId)) {...`

Reply:
LGTM

Reply:
Hi Yunfei, fyi. I have to recover to the original change and not check `if (this.tensorMap.has(dataId))`. `this.tensorMap.get(dataId)` will make sure that there is always a valid tensor. If there is not a tensor id, it will call `engine.moveData` to get one.
If we used `this.tensorMap.has(dataId)`, there will be no chance to call `engine.moveData`. In that case, we will get error in some operators.